tests/utils/queryApiGateway.test.ts

```diff
@@ -81,18 +81,18 @@ describe('Querying the API Gateway', () => {
   })
 
   it('handles errors thrown by fetch', async () => {
+    const networkErrorMessage = 'network error'
     /* eslint-disable  @typescript-eslint/no-unsafe-call */
-    fetch.mockRejectedValue(new Error('network error'))
+    fetch.mockRejectedValue(new Error(networkErrorMessage))
     /* eslint-enable  @typescript-eslint/no-unsafe-call */
 
     // Mock process.env
     const restore = mockEnv({
       API_URL: goodUrl,
     })
 
-    const data = await queryApiGateway(goodRequest)
+    await expect(queryApiGateway(goodRequest, goodUniqueNumber)).rejects.toThrow(networkErrorMessage)
```

Comment:
@lomky This is the [correct syntax](https://jestjs.io/docs/asynchronous#resolves--rejects) for testing async failures in jest.